stdlib/settings.pkl

```diff
@@ -22,41 +22,45 @@
 @ModuleInfo { minPklVersion = "0.26.0" }
 module pkl.settings
 
+import "pkl:EvaluatorSettings"
+
 /// The editor for viewing and editing Pkl files.
 editor: Editor = System
 
+http: EvaluatorSettings.Http?
+
 /// The editor associated with `file:` URLs ending in `.pkl`.
-hidden System: Editor = new {
+hidden const System: Editor = new {
   urlScheme = "%{url}, line %{line}"
 }
 
 /// The [IntelliJ IDEA](https://www.jetbrains.com/idea) editor.
-hidden Idea: Editor = new {
+hidden const Idea: Editor = new {
   urlScheme = "idea://open?file=%{path}&line=%{line}"
 }
 
 /// The [GoLand](https://www.jetbrains.com/go/) editor.
-hidden GoLand: Editor = new {
+hidden const GoLand: Editor = new {
   urlScheme = "goland://open?file=%{path}&line=%{line}"
 }
 
 /// The [TextMate](https://macromates.com) editor.
-hidden TextMate: Editor = new {
+hidden const TextMate: Editor = new {
   urlScheme = "txmt://open?url=%{url}&line=%{line}&column=%{column}"
 }
 
 /// The [Sublime Text](https://www.sublimetext.com) editor.
-hidden Sublime: Editor = new {
+hidden const Sublime: Editor = new {
   urlScheme = "subl://open?url=%{url}&line=%{line}&column=%{column}"
 }
 
 /// The [Atom](https://atom.io) editor.
-hidden Atom: Editor = new {
+hidden const Atom: Editor = new {
   urlScheme = "atom://open?url=%{url}&line=%{line}&column=%{column}"
 }
 
 /// The [Visual Studio Code](https://code.visualstudio.com) editor.
-hidden VsCode: Editor = new {
+hidden const VsCode: Editor = new {
```

Comment:
Not sure why these turned into `const`? Did I do this?

Reply:
From a previous iteration where I used a default value somewhere. Will revert (although they might as well be const, no? Minor ergonomics point; if so, other PR).